flexget/plugins/input/anidb_feed.py

```diff
@@ -0,0 +1,485 @@
+
+            # add some usefully attributes to the namespace
+            self._fill_namespace_attributes(xml_entry, new_entry)
+            if config['valid_only'] is True:
+                if self.is_valid_entry(new_entry, NAMESPACE_PREFIX) is False:
+                    continue
+            # now manually fix known fields
+            # 344.18 MB (360.895.922)
+            size_string = self.get_value_via_regex(NAMESPACE_PREFIX + 'size', new_entry, r'\((([0-9]{1,3}\.|[0-9]{1,3}){1,5})\)')
+            if size_string:
+                bytes_string = size_string.replace('.', '')
+                try:
+                    size_bytes = int(bytes_string)
+                    size_mb = int(size_bytes / 1024 / 1024)  # MB
+                    if size_mb > 0:
+                        new_entry['content_size'] = size_mb  # FIXME: is this valid here or put in NAMESPACE_PREFIX_MAIN?
+                        new_entry[NAMESPACE_PREFIX + 'size'] = size_bytes  # update with parsed size
+                except (TypeError, ValueError) as ex:
+                    log.warning('Could not extract valid size from string: %s, error: %s', size_string, ex)
+            # "HorribleSubs (HorribleSubs)"
+            group_tag = self.get_value_via_regex(NAMESPACE_PREFIX + 'group', new_entry, r'\((.*?)\)')
+            # "title - 1 - ... - Complete Movie"
+            # FIXME: 'Complete Movie' is not guaranteed?
+            is_movie = self.get_value_via_regex('title', new_entry, r'- .*?(Complete Movie)$')
+            # 1920x1080, 848x480, 1280x720
+            resolution_string = self.get_value_via_regex(NAMESPACE_PREFIX + 'resolution', new_entry, r'x([0-9]{3,4})$')
+            source_string = None
+            if new_entry.get(NAMESPACE_PREFIX + 'source'):
+                source_string = ANIDB_SOURCES_MAP.get(new_entry[NAMESPACE_PREFIX + 'source'])
+            # 'Matches official CRC (9833055b)'
+            if new_entry.get(NAMESPACE_PREFIX + 'crc_status'):
+                crc_string = self.get_value_via_regex(NAMESPACE_PREFIX + 'crc_status', new_entry,
+                                                      r'Matches official CRC \((([0-9]|[A-Fa-f]){8,8})\)')
+                if crc_string:
+                    new_entry[NAMESPACE_PREFIX_MAIN + 'file_crc32'] = crc_string.lower()
+            # "title - 6 - episode name - [group_tag]... or (344.18 MB)"
+            anidb_name = self.get_value_via_regex('title', new_entry, r'^(.*?) -')
+            anidb_name.rstrip()
+            if not anidb_name:
+                log.error('Skipping entry, invalid anidb name parsed from: %s', new_entry['title'])
+                continue
+
+            # fix and add to new_entry()
+            quality_string = ''
+            if resolution_string:
+                quality_string += resolution_string
+            if source_string:
+                quality_string += ' ' + source_string
+            if not quality_string.isspace():
```

Comment:
Won't this be true if neither `resolution_string` nor `source_string` contain a value?